### ISSUE TITLE
refinement stage

### ISSUE THREAD

**Issue by alicera** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#238]
I already the weight with refinement stage 1 
I try to train again with refinement stage 2 and it look like loss is very large. The mAP value is 0 when iteration is 5000
Is it normal ?
```
CUDA_VISIBLE_DEVICES=0,1,2,3 python train.py --train-images-folder /training/coco/train2017/ --prepared-train-labels prepared_train_annotation.pkl --val-labels val_subset.json --val-images-folder /coco/val2017/ --checkpoint-path 
 checkpoint_iter_700000.pth --weights-only --num-refinement-stages 2


NOTE! Installing ujson may make loading annotations faster.

[WARNING] Not found pre-trained parameters for refinement_stages.1.trunk.0.initial.0.weight
[WARNING] Not found pre-trained parameters for refinement_stages.1.trunk.0.initial.0.bias
[WARNING] Not found pre-trained parameters for refinement_stages.1.trunk.0.trunk.0.0.weight
[WARNING] Not found pre-trained parameters for refinement_stages.1.trunk.0.trunk.0.0.bias
[WARNING] Not found pre-trained parameters for refinement_stages.1.trunk.0.trunk.0.1.weight

```

loss
```
stage1_pafs_loss:     76.3357960510254
stage1_heatmaps_loss: 24.95510898590088
stage2_pafs_loss:     70.57485496520997
stage2_heatmaps_loss: 22.350315589904785
stage3_pafs_loss:     2214.7845043945313
stage3_heatmaps_loss: 1774.8921618652344

```

**Comment by Daniil-Osokin** [comment]
Dear @alicera, to check that everything goes well, you can do the following. Try to overfit on a small batch of data, e.g. 2-4 images. So limit a dataset to 4 images, make a batch size 4, disable augmentations, and run the training. If loss will decrease rapidly, then everything is ok. You may also visualize predicted heatmaps alongside with input images to check what was learnt and get if training goes well. Higher loss values for stages with randomly initialized weights are ok in comparison to loss values of stages with pre-trained weights.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.

**Comment by alicera** [comment]
The loss is still large.
If I change the backbone, does I have some thing to notice?
